trustmanager/keystore.go

```diff
@@ -218,31 +218,15 @@ func (s *GenericKeyStore) ListKeys() map[string]KeyInfo {
 func (s *GenericKeyStore) RemoveKey(keyID string) error {
 	s.Lock()
 	defer s.Unlock()
-	// If this is a bare key ID without the gun, prepend the gun so the filestore lookup succeeds
-	if keyInfo, ok := s.keyInfoMap[keyID]; ok {
-		keyID = filepath.Join(keyInfo.Gun, keyID)
-	}
-
-	role, legacy, err := getKeyRole(s.store, keyID)
-	if err != nil {
-		return err
-	}
-
 	delete(s.cachedKeys, keyID)
 
-	name := keyID
-	if legacy {
-		name = keyID + "_" + role
-	}
-
 	// being in a subdirectory is for backwards compatibliity
```

Comment:
Non-blocking: I think this comment is no longer relevant, since it's no longer in a subdirectory